demo/src/screens/componentScreens/CarouselScreen.js

```diff
@@ -62,6 +62,20 @@ class CarouselScreen extends Component {
         <View margin-20 center /*style={{...StyleSheet.absoluteFillObject}} */ pointerEvents="none">
           <Text text10>{this.state.currentPage}</Text>
         </View>
+
+        <View padding-20>
+          <Carousel test migrate containerStyle={{height: 160}}>
+            {_.times(5, index => {
+              return (
+                <View key={index} flex padding-20>
+                  <Card flex padding-20>
```

Comment:
I think it's not clear enough that this is a "full width" Carousel example because of the Cards...

Reply:
yea, I wanted to still look nice.
but maybe I can add a title for the example.
WDYT?

Reply:
How about something like an image gallery that looks less like the other example?